community/modules/scheduler/cloud-batch-login-node/outputs.tf

```diff
@@ -22,21 +22,15 @@ output "login_node_name" {
 output "instructions" {
   description = "Instructions for accessing the login node and submitting Google Cloud Batch jobs"
   value       = <<-EOT
-  Use the following commands to:
 
+  A Batch job template file will be placed on the Batch login node at:
+    ${local.job_template_destination}
+
+
```

Comment:
Do we need 2 blank lines here?

Reply:
It was an aesthetic choice that I do not feel strongly about. I have removed the 2nd blank line.